### PR TITLE
Search bar visuals

### PR DESCRIPTION
**Pull request by sarahscott**
This pr is just for the visuals of the search 'dummy button' and my next pr will focus on the wiring between emission & eigen.

simulator screen shot oct 11 2016 5 02 41 pm

simulator screen shot oct 11 2016 5 54 13 pm

### CODE REVIEW

File: Pod/Classes/Core/ARSwitchBoardModule.m [hunk 1]
```diff
@@ -21,6 +21,11 @@ @implementation ARSwitchBoardModule
   [self invokeCallback:self.presentModalViewController reactTag:reactTag route:route];
 }
 
+RCT_EXPORT_METHOD(presentSearchViewController:(nonnull NSNumber *)reactTag)
```
Comment (sarahscott) [inline Pod/Classes/Core/ARSwitchBoardModule.m]: I know that all of these will have to be in the pod and not here; this is more of a sketch of what will need to happen later. All of this should be gone by the time this particular PR is mergeable.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">No CHANGELOG added.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">The following files do not have tests: <a href="https://github.com/artsy/emission/blob/b0bb5c4699a22f25f93682c7fe0cbe7b7474c8d5/lib/components/home/search_bar.js">lib/components/home/search_bar.js</a>, <a href="https://github.com/artsy/emission/blob/b0bb5c4699a22f25f93682c7fe0cbe7b7474c8d5/lib/containers/home.js">lib/containers/home.js</a> &amp; <a href="https://github.com/artsy/emission/blob/b0bb5c4699a22f25f93682c7fe0cbe7b7474c8d5/lib/native_modules/switch_board.js">lib/native_modules/switch_board.js</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: Pod/Classes/Core/ARSwitchBoardModule.m (hunk 1 above)
Comment (sarahscott) [inline Pod/Classes/Core/ARSwitchBoardModule.m]: These have since been removed

**Comment by sarahscott** [comment]
@mennenia ready for review!

**Comment by alloy** [comment]
I’m taking this :+1:

@sarahscott The search icon assets for 1x and 3x are missing, please don’t forget to add those!

**Pull request merged**